src/routes/exercise-finished/ExerciseFinishedScreen.tsx

```diff
@@ -4,14 +4,22 @@ import React, { ReactElement } from 'react'
 import { heightPercentageToDP as hp, widthPercentageToDP as wp } from 'react-native-responsive-screen'
 import styled from 'styled-components/native'
 
-import { CheckCircleIconWhite, ListIcon, RepeatIcon } from '../../../assets/images'
+import {
+  CheckCircleIconWhite,
+  ListIcon,
+  RepeatIcon,
+  UnlockLockIcon,
+  CloseIcon,
+  CloseIconWhite
+} from '../../../assets/images'
 import Button from '../../components/Button'
 import { HeadingBackground } from '../../components/text/Heading'
 import { BUTTONS_THEME, ExerciseKeys, EXERCISES } from '../../constants/data'
 import labels from '../../constants/labels.json'
 import { RoutesParams } from '../../navigation/NavigationTypes'
 import ShareSection from './components/ShareSection'
 
+const ExerciseUnlocked = false // here is a the boolean should be replaced with the logic
```

Comment:
Please also use camelCase here for naming.
Please also link the logic issue here, just add `LUN-131` to the comment, also please do above in ExercisesScreen in the comment.